src/Three.js

```diff
@@ -116,10 +102,8 @@ export { AnimationMixer } from './animation/AnimationMixer.js';
 export { AnimationClip } from './animation/AnimationClip.js';
 export { AnimationAction } from './animation/AnimationAction.js';
 export { Uniform } from './core/Uniform.js';
-export { InstancedBufferGeometry } from './core/InstancedBufferGeometry.js';
-export { BufferGeometry } from './core/BufferGeometry.js';
-export { DirectGeometry } from './core/DirectGeometry.js';
-export { GeometryIdCount, Geometry } from './core/Geometry.js';
+export * from './core/Geometries.js';
```

Comment:
I like `./materials/Material.js` and `./extras/geometries/Geometries.js` but I wouldn't this `./core/Geometries.js` feels like over-engineering to me.